conda.recipe/meta.yaml

```diff
@@ -8,10 +8,10 @@ build:
 
 requirements:
   build:
-    - "python<3.12"
-    - "numpy>=1.14"
-    - "pandas>=1.2.0"
-    - "bokeh>=1.4.0, <3.0.0"
+    - "python>=3.9, <3.12"
+    - "numpy>=1.20, <2.0"
```

Comment:
Is it important to provide an upper bound on the versions for certain packages, like `numpy` and `pandas`?  Historically, Tax-Calculator has tried to remain compatible with the most recent versions of the dependencies.